src/main/java/org/jenkinsci/plugins/mesos/MesosAgent.java

```diff
@@ -30,9 +30,9 @@
 import org.slf4j.LoggerFactory;
 
 /** Representation of a Jenkins node on Mesos. */
-public class MesosSlave extends AbstractCloudSlave implements EphemeralNode {
+public class MesosAgent extends AbstractCloudSlave implements EphemeralNode {
```

Comment:
maybe MesosJenkinsAgent is a better name? I suppose the name MesosAgent gets confusing in the context of mesos

Reply:
I renamed it but we also have `MesosAgentSpecTemplate`. We probably need a few more iterations on naming 🙂